examples/composition/configuration.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: meta.pkg.crossplane.io/v1alpha1
+kind: Configuration
```

Comment:
If you would like to set up building and publishing this `Configuration` for folks to be able to install we can add a workflow for that :)

Reply:
@hasheddan yes, that would be great !